src/components/Common/Combobox.tsx

```diff
@@ -142,7 +142,7 @@ export const Combobox = ({
         </div>
       </div>
       <ul
-        className={`z-10 max-h-80 overflow-auto rounded bg-gray-100 dark:bg-gray-700 ${
+        className={`z-10 max-h-80 overflow-auto rounded bg-layer-0 ${
```

Comment:
```suggestion
        className={`z-10 max-h-80 overflow-auto rounded bg-layer-3 ${
```
![image](https://github.com/epam/ai-dial-chat/assets/20163971/79904567-113f-486d-b503-5500be2499d9)